src/skin/tooltips.cpp

```diff
@@ -783,7 +794,7 @@ void Tooltips::addStandardTooltips() {
 
     add("EffectSlot_enabled")
             << tr("Enable Effect")
-            << tr("This effect chain must also be assigned to a deck or other sound source to hear the effect.");
+            << tr("This effect unit must also be assigned to a deck or other sound source to hear the effect.");
```

Comment:
The effect unit
(The effect unit this effect is part of)

Reply:
good point

Reply:
Ok, I will fix this after merge.